Dockerfile

```
@@ -0,0 +1,17 @@
+FROM python:3.8.8-buster AS base
+RUN apt-get update && apt-get install -y curl
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+EXPOSE 5000
+ENV PATH="${PATH}:/root/.poetry/bin"
+RUN mkdir -p /devops_mod5
+WORKDIR /devops_mod5
+COPY . /devops_mod5
+RUN poetry install 
+ENV VIRTUAL_ENV=.venv
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+FROM base AS production
+ENTRYPOINT ["gunicorn", "-w", "3", "-b", "0.0.0.0:5000", "todo_app.app:app"]
```

Why have you got rid of your create_app function in app.py? You can use a function in your gunicorn command:

```suggestion
ENTRYPOINT ["gunicorn", "-w", "3", "-b", "0.0.0.0:5000", "todo_app.app:create_app()"]
```

---

That was the first thing I tried but it wouldn't work. I wasted a lot of time trying to get it work so I removed it.
Just realised all I needed to do was to add "()" to `create_app` when running the gunicorn command

Thanks a lot for this Hugh! Won't need to rewrite my tests now :)